src/main/java/core/basesyntax/JavaDateTimeApi.java

```diff
@@ -19,7 +36,19 @@
      * In any other case throw DateTimeException.
      **/
     public String todayDate(DateTimePart datePart) {
-        return "Today";
+        if (datePart == DateTimePart.YEAR) {
+            return String.valueOf(LocalDate.now().getYear());
+        }
+        if (datePart == DateTimePart.MONTH) {
+            return LocalDate.now().getMonth().toString();
+        }
+        if (datePart == DateTimePart.DAY) {
+            return String.valueOf(LocalDate.now().getDayOfMonth());
+        }
+        if (datePart == DateTimePart.FULL) {
+            return LocalDate.now().toString();
+        }
```

Comment:
better to initialize LocalDate.now at the beginning and then just take out what you want.
should use switch in this case. your code would be more readability